addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -51,9 +51,14 @@ def test_channel_member_cannot_be_public_user(self):
     @users('employee')
     @freeze_time("2020-03-22 10:42:06")
     def test_channel_members(self):
-        channel = self.env['discuss.channel'].browse(self.test_channel.ids)
-        self.assertEqual(channel.message_partner_ids, self.env['res.partner'])
-        self.assertEqual(channel.channel_partner_ids, self.env['res.partner'])
+        test_group = self.env["discuss.channel"].create(
+            {
+                "name": "Group",
+                "channel_type": "group",
+            }
+        )
```

Comment:
can be single line here

```suggestion
        test_group = self.env["discuss.channel"].create({"name": "Group", "channel_type": "group"})
```